core/src/helpers/formatToAutomaticDecimalPoints.ts

```diff
@@ -58,11 +58,25 @@ function limitedValue(value: number | BigNumber): number | BigNumber {
     return value;
 }
 
+// We choose regex over native JS solutions, as using native solutions often misformatted the decimal points, leading to inconsistent number formatting
```

Comment:
This comment is strange and is not the same as in https://github.com/sidestream-tech/unified-auctions-ui/pull/481/files#r983341267. There the main point there is: `native functions only support the number type`, here: `native solutions often misformatted the decimal points` – which doesn't even sounds convincing. What do you mean by `often misformatted`?

Reply:
I updated the comment to be more inline!